org.upbge.UPBGE.yml

```
@@ -0,0 +1,79 @@
+id: org.upbge.UPBGE
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: blender
+finish-args:
+  - "--share=ipc"
+  - "--socket=wayland"
+  - "--socket=fallback-x11"
+  - "--socket=pulseaudio"
+  - "--device=dri"
+  - "--share=network"
+  - "--filesystem=host"
+  - "--filesystem=/run/spnav.sock:ro"
+  - "--env=SPNAV_SOCKET=/run/spnav.sock"
+  - "--env=TMP_DIR=/tmp"
+  - "--env=TMP=/tmp"
+add-extensions:
+  org.upbge.UPBGE.Codecs:
+    directory: lib/codecs
+    add-ld-path: lib
+    bundle: true
+    autodelete: true
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '22.08'
```

this should match the runtime version

```suggestion
    version: '23.08'
```